src/main/kotlin/com/fairchild_superconductor/magic_crystal/MagicCrystal.kt

```diff
@@ -5,13 +5,25 @@ import com.fairchild_superconductor.magic_crystal.computer.ComputerEntity
 import net.fabricmc.api.ModInitializer
 import net.fabricmc.fabric.api.`object`.builder.v1.block.FabricBlockSettings
 import net.fabricmc.fabric.api.`object`.builder.v1.block.entity.FabricBlockEntityTypeBuilder
+import net.fabricmc.fabric.api.biome.v1.BiomeModifications
+import net.fabricmc.fabric.api.biome.v1.BiomeSelectors
```

Comment:
Your IDE may say this is `Deprecated`, but in fact it is `Experimental`.
I don't think Fabric author use `@Deprecated` correctly.